api/models/build-task.js

```diff
@@ -0,0 +1,56 @@
+const { buildEnum } = require('../utils');
+
+const Statuses = buildEnum([
+  'created',
+  'queued',
+  'tasked',
+  'error',
+  'processing',
+  'skipped', // remove?
+  'success',
+]);
+
+const associate = ({ BuildTask, Build }) => {
+  BuildTask.belongsTo(Build, {
+    foreignKey: 'buildId',
+    allowNull: false,
+  });
+};
+
+module.exports = (sequelize, DataTypes) => {
+  const BuildTask = sequelize.define(
+    'BuildTask',
+    {
+      name: {
+        type: DataTypes.STRING,
+        allowNull: false,
+      },
+      artifact: {
+        type: DataTypes.STRING,
+      },
+      status: {
+        type: DataTypes.ENUM,
+        values: Statuses.values,
+        defaultValue: Statuses.Created,
+        allowNull: false,
+        validate: {
+          isIn: [Statuses.values],
+        },
+      },
+    }, {
+      tableName: 'build_task',
+      paranoid: true,
+      indexes: [
+        {
+          name: 'build_task_build_id_type_index',
+          unique: true,
+          fields: ['buildId'],
```

Comment:
You need to add `buildTaskTypeId` based on the migration SQL.

Reply:
Excellent catch. My kingdom for not having to write these schemas twice